Volume/Views/WeeklyDebriefView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  WeeklyDebriefView.swift
+//  Volume
+//
+//  Created by Amy Chin Siu Huang on 12/1/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct WeeklyDebriefView: View {
+    @EnvironmentObject private var userData: UserData
+    let weeklyDebrief: WeeklyDebrief
+    @Binding var openedWeeklyDebrief: Bool
+
+    init(openedWeeklyDebrief: Binding<Bool>, weeklyDebrief: WeeklyDebrief) {
+        UIPageControl.appearance().currentPageIndicatorTintColor = .gray
+        UIPageControl.appearance().pageIndicatorTintColor = .lightGray
```

Comment:
Do we ever set this back to what it was before (e.g., when you leave this view)? Also, I think this should got in `onAppear`